tests/integration/endpoint_test.go

```diff
@@ -14,17 +14,24 @@ func endpointTests(t *testing.T, when spec.G, it spec.S) {
 	var service testutil.TestService
 	var stagedService testutil.TestService
 
+	it.Before(func() {
+		service.Create(t, "ibuildthecloud/demo:v1")
+		stagedService = service.Stage("ibuildthecloud/demo:v3", "v3")
+	})
 	it.After(func() {
 		service.Remove()
 		stagedService.Remove()
 	})
 
 	when("a service is running and another is staged", func() {
 		it("should have endpoints that are available on both", func() {
-			service.Create(t, "ibuildthecloud/demo:v1")
-			stagedService = service.Stage("ibuildthecloud/demo:v3", "v3")
-			assert.Equal(t, "Hello World", service.GetEndpoint())
+						assert.Equal(t, "Hello World", service.GetEndpoint())
 			assert.Equal(t, "Hello World v3", stagedService.GetEndpoint())
+			assert.Equal(t, service.GetKubeEndpointUrlByRevision("v0"), service.GetEndpointUrl())
```

Comment:
Should we test against the wanted "hello world" string instead ? If both endpoints return nothing or a 500, are we ok with that as long as they are the same ? I know we do a check above that makes that question void but it's possible in the future it gets removed for some reason.